src/color-theme/style.css

```
@@ -0,0 +1,498 @@
+@layer reset, base, theme, layout;
+
+@property --header-hue {
+  syntax: '<number>';
+  inherits: false;
+  initial-value: 100;
+}
+
+@property --base-color-light {
+  syntax: '<color>';
+  inherits: false;
+  initial-value: oklch(43.7% 0.075 224);
+}
+
+@property --base-color-dark {
+  syntax: '<color>';
+  inherits: false;
+  initial-value: oklch(89.2% 0.069 224);
+}
+
+
+
+
+[data-playlist-item] {
+  align-items: center;
+  border-inline-start: max(2cqi, 2ch) solid color-mix(in oklab, var(--song-color) 80%, var(--background-color));
+  column-gap: calc(var(--gap) / 4);
+  display: grid;
+  grid-template:
+    "song" auto
+    "artist" auto
+    "rating" auto/ 100%;
+  font-size: var(--text-small);
+  justify-items: start;
+  padding: calc(var(--gap) / 2);
+
+  @container (min-width: 30em) {
+    grid-template:
+      "song rating" auto
+      "artist artist" auto / minmax(0, max-content) max-content;
```

Otherwise a song title, like Yellow Days goes onto two lines at some widths

```suggestion
  "artist artist" auto / minmax(0, auto) 1fr;
```